README.md

```diff
@@ -39,7 +39,21 @@ This is all possible inside `todo.txt`.
 
 ## `todo.txt` format rules
 
-<img src="./description.svg" width="100%" height="500">
+```
+┌─────────────────── Optional - Marks completion 
+│  ┌──────────────── Optional - Marks priority 
+│  │      ┌─────────────── Optional - Completion date 
+│  │      │          ┌──── Optional - Creation date
+│  │      │          │     (must be specified if completion date is)
+│  │      │          │             ┌─── Mandatory - Description
+│  │      │          │             │    Tags (optional) can be placed anywhere within it.
+│  │  ╭───┴───╮  ╭───┴───╮  ╭──────┴───────────────────────────────────────────────╮
+x (A) 2016-05-20 2016-04-30 measure space for +chapelShelving @chapel due:2016-05-30
+                                              └──────┬──────┘ └──┬──┘ └──────┬─────┘       
+                                    project tag ─────┘           │           │
+                                    context tag ─────────────────┘           │
+                                    special keyvalue tag ────────────────────┘
```

Comment:
It's more common to spell this as "key-value" (with a hyphen):

```suggestion
                                    special key-value tag ───────────────────┘
```

Reply:
Also, I would suggest using a word other than "special", which implies that there's a limited set of valid such tags, whereas this is actually the opposite. Something like "arbitrary" would convey the meaning more clearly, IMO.